cdap-ui/cypress/integration/pipeline.upgrade.spec.ts

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright © 2019 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import * as Helpers from '../helpers';
+
+describe('Pipeline Upgrade should work fine', () => {
```

Comment:
My worry about this is that we have to maintain the versions of the plugins manually... not sure what's the best alternative approach though :(

Reply:
Agreed. As mentioned in the https://github.com/cdapio/cdap/pull/10935#discussion_r246570847 lets fix the UI to handle artifact version range or latest artifact for us to be able to avoid updating this every release.